scripts/core/editor3/components/Editor3.jsx

```diff
@@ -0,0 +1,224 @@
+        this.onChange(
+            RichUtils.toggleBlockType(
+                this.state.editorState,
+                blockType
+            )
+        );
+    }
+
+    _toggleInlineStyle(inlineStyle) {
+        this.onChange(
+            RichUtils.toggleInlineStyle(
+                this.state.editorState,
+                inlineStyle
+            )
+        );
+    }
+
+    render() {
+        const {editorState} = this.state;
+
+        // If the user changes block type before entering any text, we can
+        // either style the placeholder or hide it. Let's just hide it now.
+        let className = 'Editor3-editor';
+        var contentState = editorState.getCurrentContent();
+        if (!contentState.hasText()) {
+            if (contentState.getBlockMap().first().getType() !== 'unstyled') {
+                className += ' Editor3-hidePlaceholder';
+            }
+        }
+
+        if (this.showToolbar) {
+            return (
+                <div className="Editor3-root">
+                    <div className="Editor3-controls">
+                        <BlockStyleControls
+                            editorState={editorState}
+                            options={this.editorFormat}
+                            onToggle={this.toggleBlockType}
+                        />
+                        <InlineStyleControls
+                            editorState={editorState}
+                            options={this.editorFormat}
+                            onToggle={this.toggleInlineStyle}
+                        />
+                    </div>
+                    <div className={className} onClick={this.focus}>
+                        <Editor
+                            blockStyleFn={getBlockStyle}
+                            customStyleMap={styleMap}
+                            editorState={editorState}
+                            handleKeyCommand={this.handleKeyCommand}
+                            onChange={this.onChange}
+                            onTab={this.onTab}
+                            readOnly={this.readOnly}
+                            ref="editor"
+                        />
+                    </div>
+                </div>
+            );
+        } else {
+                return (
+                    <div onClick={this.focus} className="Editor3-editor-single-line">
+                        <Editor
+                            blockStyleFn={getBlockStyle}
+                            customStyleMap={styleMap}
+                            editorState={editorState}
+                            handleKeyCommand={this.handleKeyCommand}
+                            onChange={this.onChange}
+                            onTab={this.onTab}
+                            readOnly={this.readOnly}
+                            ref="editor"
+                        />
+                    </div>
+            );
+        }
+    }
+}
+
+// Custom overrides for "code" style.
+const styleMap = {
```

Comment:
As you created a css file for the editor3, is it possible to put all the style there ?